src/backends/DomRenderer.js

```diff
@@ -67,6 +67,17 @@ const getHead = (() => {
 })()
 
 /**
+ * Gets the `body` element upon the first call and caches it.
+ */
+const getBody = (() => {
+  let body
+  return (): HTMLElement => {
+    if (!body) body = document.body || document.getElementsByTagName('body')[0]
```

Comment:
do you know which browsers still need a fallback to `getElementsByTagName`, if we target ie9+ maybe this can be omited already also for the head mb? It would mean we don't need to cache those.